README.md

```diff
@@ -25,26 +25,29 @@ See https://github.com/fossology/fossology/releases for source code download of
 
 For installation instructions see [Github Wiki](https://github.com/fossology/fossology/wiki)
 
-## Docker
-
-FOSSology comes with a Dockerfile allowing the containerized execution
-both as single instance or in combination with an external PostgreSQL database.
+### Install as a single container with all services combined
 **Note:** It is strongly recommended to use an external database for production
-use, since the the standalone image does not take care of data persistency.
+use, since the the single container does not handle data persistence.
+
+FOSSology comes with a [Dockerfile](https://github.com/fossology/fossology/blob/master/Dockerfile) which creates a single container which runs all three services; web-based UI, the FOSSology scheduler, and the PostgresSQL database. A pre-built Docker image is available from [Docker Hub](https://hub.docker.com/r/fossology/fossology/) and can be run using following commands:
 
-A pre-built Docker image is available from [Docker Hub](https://hub.docker.com/r/fossology/fossology/) and can be run using following command:
 ``` sh
 docker run -p 8081:80 fossology/fossology
 ```
+The docker image's web-base UI can then be used viewed http://IP_OF_DOCKER_HOST:8081/repo user fossy password fossy.
 
-The docker image can then be used using http://IP_OF_DOCKER_HOST:8081/repo user fossy passwd fossy.
-
-Execution with external database container can be done using Docker Compose, via the following command:
+### Install separate services with docker-compose
```

Comment:
It might be worth noting, that here the same image is used, as in the single container variant.

Reply:
Yeah, that might be a good idea. But are the images identical? It appears that they are both built by the same Dockerfile but I thought there was some separate configuration, I'll have to dig deeper to understand more.

Reply:
The Image is just build from the dockerfile, no configuration there. One could also start the docker-compose setup with the released image on dockerhub.

Reply:
But the instructions do not describe or support that separate process. Don't you think the instructions should be as clear as possible an support limited, well-defined, supported processes? This reduces the out-of-process installs and improves UX greatly.